lib/Service/CircleHelper.php

```diff
@@ -260,6 +260,37 @@ public function getLevel(string $circleId, string $userId): int {
 		return $member->getLevel();
 	}
 
+	/**
+	 * @throws NotFoundException
+	 * @throws NotPermittedException
+	 * @throws MissingDependencyException
+	 */
+	public function canLeave(string $circleId, string $userId): bool {
+		if (is_null($this->circlesManager)) {
+			throw new MissingDependencyException($this->dependencyInjectionError);
+		}
+
+		try {
+			$this->startSession($userId);
+			$circle = $this->circlesManager->getCircle($circleId);
+			$initiator = $circle->getInitiator();
+			if ($initiator->getUserType() !== Member::TYPE_USER) {
+				return false;
+			}
+			$members = $circle->getMembers();
+		} catch (CircleNotFoundException $e) {
+			throw new NotFoundException($e->getMessage(), 0, $e);
+		}
+
+		foreach ($members as $member) {
+			if ($member->getSingleId() !== $initiator->getSingleId()) {
+				return true;
+			}
```

Comment:
This returns true with the first user that does not match the initiator. Is this intended?

Reply:
Exactly. Because this means that at least one other member exists in the circle, so we can leave it.